<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRegistros">
     <attribute name="title">
      <string>Registros</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sala:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSalasRegistro">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aluno</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAlunoRegistro">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>191</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Número de chaves Disponiveis:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Emprestar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Emprestar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Devolver">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Devolver</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetRegistros">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>721</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Sala</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Aluno</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hora Emprestimo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hora Devolução</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAlunos">
     <attribute name="title">
      <string>Alunos</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Curso:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricula:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAlunoNome">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCurso">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>281</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sistemas de Informação</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Licenciatura em Computação</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Engenharia Ambiental</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Engenharia Cartográfica e Agrimensura</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agronomia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Engenharia de Pesca</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Engenharia Florestal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medicina Veterinária</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Zootecnia</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEditMatricula">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTelefone">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdicionarAluno">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemoverAluno">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetAlunos">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>241</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEditarAluno">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonVerDados">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Dados</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProfessores">
     <attribute name="title">
      <string>Professores</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProfNome">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProfTelefone">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProfEmail">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetProf">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>241</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonProfAdicionar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonProfRemover">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonProfEditar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonProfVerDados">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Dados</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSalas">
     <attribute name="title">
      <string>Salas</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identificacao da Sala:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alunos </string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetSalaAluno">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddAlunoSala">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Aluno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIdSala">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetSalaProf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>171</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Professores </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alunos associados</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetSalaAlunoAssociados">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddProfSala">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Professor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDelAlunoSala">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>460</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de Chaves:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdicionarSala">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Sala</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNumeroChaves">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetSalaProfAssociados">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Professores associados</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDelProfSala">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidgetSalasCadastradas">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>191</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Salas Cadastradas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionExportar_Relatorio"/>
    <addaction name="actionCarregar"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionExportar_Relatorio">
   <property name="text">
    <string>Exportar Relatorio</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
